tests/code_size/random_printf_wasm2js.json

```diff
@@ -1,6 +1,6 @@
 {
-  "a.html": 17794,
-  "a.html.gz": 7593,
-  "total": 17794,
-  "total_gz": 7593
+  "a.html": 17804,
```

Comment:
This is the only code size that increases, but when I locally do a build of this test case on current `main` vs this branch, I get exactly identical results to the byte, so I wonder if the old numbers weren't up to date..

Reply:
Indeed, there is some tolerance in the test code so the old number are rarely 100% correct.

What I normally do when I really want to see the impact of a change is run `./tests/runner other.*code_size* other.*metadce* --rebase` on `origin/main` and submit that change before doing the same on my work branch.

You can see several such changes in the git history where we just bring the current numbers up-to-date. e.g: de4cb659f9653777d2bdb33c44aab73f65749c74

Reply:
Looks like in the latest the code size here also decreases, adjusted to do optimization a bit wider. Also restored the code size test on Windows and macOS, they do seem to currently agree with each other.

Reply:
Thats great news!   I was worried that debugging that would be pain